recipes/set-timezone.rb

```diff
@@ -3,8 +3,14 @@
 
 timezone = node.fetch(:timezone, 'America/New_York')
 
+# need to create a service stub to notify
+service 'rsyslog' do
+  action :nothing
+end
+
 execute 'set timezone' do
   command %Q|timedatectl set-timezone "#{timezone}"|
   retries 5
   retry_delay 5
+  notifies :restart, "service[rsyslog]", :immediately
```

Comment:
without this the syslog timestamps keep using the old timezone (which makes the cloudwatch log agent complain & fail)